src/binary/uint.rs

```diff
@@ -47,7 +47,7 @@ impl UInt {
     }
```

Comment:
In this file:
* A previous commit had added write support for [the low-level primitive `UInt`](https://amzn.github.io/ion-docs/docs/binary.html#uint-and-int-fields). However, the method name didn't follow the naming convention used elsewhere, which refers to the Rust data type being accepted as a value to write out. `UInt::write_uint` is thus being renamed `UInt::write_u64`.